draft-ietf-scitt-architecture.md

```diff
@@ -371,6 +371,156 @@ Issuers SHOULD use consistent decentralized identifiers for all their Statements
 They MAY update their DID manifest, for instance to refresh their signing keys or algorithms, but they SHOULD NOT remove or change any prior keys unless they intend to revoke all Signed Statements that are registered as Transparent Statements issued with those keys.
 This DID appears in the Issuer protected header of Signed Statements' Envelopes, while the version of the key from the manifest used to sign the Signed Statement is written in the `kid` header.
 
+`kid` MUST either be an absolute URL,
+or `iss` MUST also be present in the protected header,
+and `kid` MUST be a URL relative to `iss`.
```

Comment:
Since `iss` seems to be required to be a DID ([this reference?](https://datatracker.ietf.org/doc/html/draft-ietf-scitt-architecture-01#section-6.1-4.2)), would it be clearer to say absolute/relative DID URL?